client/components/wcc-settings-form/settings-group.js

```diff
@@ -98,6 +104,34 @@ const SettingsGroup = ( props ) => {
 				</div>
 			);
 
+		case 'summary':
+			const renderStepSummary = ( step, stepIndex ) => (
+				<div key={ stepIndex } className="settings-step-summary">
+					<h4>{ step.tab_title }</h4>
+					<Summary
+						formValues={ form.values }
+						layoutItems={ step.items }
+						summaryTemplate={ step.summary }
+						storeOptions={ storeOptions }
+						fieldsOptions={ form.fieldsOptions } />
+					<Button compact
+							onClick={ () => formActions.goToStep( stepIndex ) } >
+						{ __( 'Edit' ) }
+					</Button>
+				</div>
+			);
+
+			return (
+				<div className="settings-group-default">
+					<FormSectionHeading dangerouslySetInnerHTML={ sanitizeHTML( group.title ) } />
```

Comment:
I just tested my snippet, and it worked as I expected it to:

<img width="375" alt="screen shot 2016-08-08 at 5 05 04 pm" src="https://cloud.githubusercontent.com/assets/63922/17499373/594d5624-5d8a-11e6-86c0-23650b8b10c4.png">